src/wizards/Wizard.tsx

```diff
@@ -0,0 +1,415 @@
+
+        if (this.state.currentStep == step.stepId) classNames.push(ClassNames.ACTIVE);
+        if (this.state.allSteps[step.stepId].stepCompleted) classNames.push(ClassNames.COMPLETE);
+        if (this.props.navLinkClasses) classNames.push(this.props.navLinkClasses);
+        // TODO: Add class error if step is not valid
+        return classNames;
+    }
+
+    // Check validity of given step
+    checkStepValidity(step: WizardStep) {
+        let validationState = true;
+        if (this.state.allSteps[step.stepId].isStepValid !== undefined)
+            validationState = this.state.allSteps[step.stepId].isStepValid!();
+
+        this.setState({
+            allSteps: [...this.state.allSteps, (this.state.allSteps[step.stepId].stepCompleted = validationState)],
+        });
+
+        return validationState;
+    }
+
+    // Build DOM for Wizard footer
+    buildWizardFooter(): React.ReactElement {
+        const {cancelButtonText, nextButtonText, previousButtonText, finishButtonText} = this.props;
+
+        return (
+            <div className={ClassNames.WIZARD_FOOTER}>
+                <div className={ClassNames.WIZARD_FOOTER_BUTTON}>
+                    <Button key={cancelButtonText} link onClick={this.close.bind(this)}>
+                        {cancelButtonText}{" "}
+                    </Button>
+
+                    {this.state.showPreviousButton && (
+                        <Button key={previousButtonText} onClick={this.previousButtonClick.bind(this)}>
+                            {previousButtonText}{" "}
+                        </Button>
+                    )}
+
+                    {this.state.showNextButton && (
+                        <Button key={nextButtonText} primary onClick={this.nextButtonClick.bind(this)}>
+                            {nextButtonText}{" "}
+                        </Button>
+                    )}
+
+                    {this.state.showFinishButton && (
+                        <Button
+                            key={finishButtonText}
+                            state={ButtonState.SUCCESS}
+                            onClick={this.finishButtonClick.bind(this)}
+                        >
+                            {finishButtonText}
+                        </Button>
+                    )}
+                </div>{" "}
+                {/*Close modal-footer-buttons*/}
+                {/*Close modal-footer*/}
+            </div>
+        );
+    }
+
+    // Build DOM for Wizard NAV
+    buildWizardNav(): React.ReactElement {
+        const {steps, title, showNav} = this.props;
+
+        return (
+            <VerticalNav className={ClassNames.WIZARD_STEPNAV_WRAPPER}>
+                <h3 className={ClassNames.WIZARD_TITLE}> {title} </h3>
+                {showNav && (
+                    <div className={ClassNames.WIZARD_STEPNAV}>
+                        {steps.map((step, key) => {
+                            return (
+                                <NavLink
```

Comment:
When I inspect the html from VMWare Clarity, I can see it's using <button/> instead of <a/>.  It might be more appropriate for us to return something like: `<Button link={true}, icon={step.customStepNav.stepNavIcon} className="clr-wizard-stepnav-link">`  If we do that, we might not need any changes in NavLink.tsx.

Reply:
I tried to use our Button component as you suggested but it was causing CSS issues for Wizard Navigation.

Reply:
As discussed changed this code to use 'Button'.